Task_10/truck_delivery_rayus007/client_manager.py

```diff
@@ -0,0 +1,28 @@
+import json
+from truck_delivery_rayus007.util.constants import UID, CLIENT
+from truck_delivery_rayus007.db_connector_redis import DbConnectorRedis
+from truck_delivery_rayus007.content_manager import ContentManager
+
+
+class ClientManager(ContentManager):
+    """ Class representing a ClientManager """
+    def __init__(self):
+        super().__init__()
+        self.connector = DbConnectorRedis()
+
+    def save_document(self, key, document):
+        result_save = self.connector.save(f"{UID}_{CLIENT}_{key}", json.dumps(document))
```

Comment:
you should be receiving a client Object here and saving that

Reply:
I am getting/saving a Client object now